sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdServiceEndpoint.java

```diff
@@ -379,7 +379,8 @@ private RntbdRequestRecord writeWhenConnected(
             assert channel != null : "impossible";
             this.releaseToPool(channel);
```

Comment:
we are releasing twice , this line needs to be deleted ?

Reply:
oops, yea, I thought I removed it